<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmImagePilot</class>
 <widget class="QWidget" name="frmImagePilot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="ImagePilot" name="imagePilot1" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>157</width>
     <height>138</height>
    </rect>
   </property>
  </widget>
  <widget class="ImagePilot" name="imagePilot2" native="true">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>9</y>
     <width>156</width>
     <height>138</height>
    </rect>
   </property>
  </widget>
  <widget class="ImagePilot" name="imagePilot3" native="true">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>9</y>
     <width>157</width>
     <height>138</height>
    </rect>
   </property>
  </widget>
  <widget class="ImagePilot" name="imagePilot4" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>153</y>
     <width>157</width>
     <height>138</height>
    </rect>
   </property>
  </widget>
  <widget class="ImagePilot" name="imagePilot5" native="true">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>153</y>
     <width>156</width>
     <height>138</height>
    </rect>
   </property>
  </widget>
  <widget class="ImagePilot" name="imagePilot6" native="true">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>153</y>
     <width>157</width>
     <height>138</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImagePilot</class>
   <extends>QWidget</extends>
   <header>imagepilot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
